src/sbd-cluster.c

```diff
@@ -308,7 +308,9 @@ sbd_membership_connect(void)
         }
     }
 
-    set_servant_health(pcmk_health_transient, LOG_NOTICE, "Connected, waiting for initial membership");
+    set_servant_health(pcmk_health_transient, LOG_NOTICE,
```

Comment:
Not sure if it still makes sense but this is outside of SUPPORT_COROSYNC section which
means that two_node might not be there.